pkg/ocm/sca/architectures_gather.go

```diff
@@ -0,0 +1,47 @@
+package sca
+
+import (
+	"context"
+	"runtime"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Mapping of kubernetes architecture labels to the format used by SCA API
+var kubernetesArchMapping = map[string]string{
+	"386":     "x86",
+	"amd64":   "x86_64",
+	"ppc":     "ppc",
+	"ppc64":   "ppc64",
```

Comment:
```suggestion
	"ppc":     "ppc",
	"ppc64":   "ppc64",
```

only ppc64le is supported. There are no plans to support Big Endian

Reply:
I have used all architectures that are supported by AMS api. But you are right that OCM right now supports only 4 architectures.
It was also discussed here: https://docs.google.com/document/d/1kT8uzjbmTTN2Zyhfo1jhMNR8q3PhkNJ9yGx83FOFL8k/edit?pli=1&disco=AAABZjOhM1A